honeybee_radiance/dynamic/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Objects for creating and exporting dynamic geometry."""
+
+from .group import DynamicShadeGroup, DynamicSubFaceGroup
+from .state import RadianceShadeState, RadianceSubFaceState
```

Comment:
If they are Radiance only should we remove Radiance form in front of the names? Basically call the `ShadeState` and `SubFaceState`?

Reply:
I am a bit uncertain about whether we are going to create similar objects over on honeybee-energy, in which case we could have conflicting names of classes in the schema. It's possible that I end up settling on a different organization of objects over there but it seems safer to put `Radiance` in the name for now.